### PR TITLE
fix: create empty array of options if undefined

### LINKED ISSUE #8: Bug: Loop on options crash when no options
```
jenklab-ci/src/index.js:184
            buildJobRequest(args.job, options.parameter.map(descriptor => new Parameter(descriptor)))
                                                       ^

TypeError: Cannot read property 'map' of undefined
    at Command.caporal.name.version.description.command.argument.option.option.option.option.option.option.option.action [as _action] (/jenklab-ci/src/index.js:184:56)
    at Command._run (/jenklab-ci/node_modules/caporal/lib/command.js:407:40)
    at Program._run (/jenklab-ci/node_modules/caporal/lib/program.js:151:16)
    at Program.parse (/jenklab-ci/node_modules/caporal/lib/program.js:254:17)
    at Object.<anonymous> (/jenklab-ci/src/index.js:211:9)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/jenklab-ci/bin/jenklab-ci:5:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

### PR DESCRIPTION
**Pull request by desaintvincent**
create an empty array if there is no options to prevent crash.

close #8

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -181,7 +181,13 @@ caporal
                 crumbIssuer: true,
             });
 
-            buildJobRequest(args.job, options.parameter.map(descriptor => new Parameter(descriptor)))
+            let optionsParameter = [];
```
Comment (wixyvir) [inline src/index.js]: Why not a simple conditional ternary operator?

Comment (desaintvincent) [reply src/index.js]: done

**Comment by wixyvir** [comment]
:+1:

File: src/index.js [hunk 2]
```diff
@@ -181,7 +181,8 @@ caporal
                 crumbIssuer: true,
             });
 
-            buildJobRequest(args.job, options.parameter.map(descriptor => new Parameter(descriptor)))
+            buildJobRequest(args.job, options.parameter === undefined ? [] :
```
Comment (jubianchi) [inline src/index.js]: ˋ(options.parameter || []).map(...)`

**Review by jubianchi — CHANGES_REQUESTED** [review]

**Review by jubianchi — APPROVED** [review]

**Comment by jubianchi** [comment]
Closed in favor of #10

**Pull request closed without merging**